### DISCUSSION TITLE
How to include all bib entries in distill posts without citing them in the post?

### DISCUSSION THREAD

**Discussion by rohandebsarkar** [alshedivat/al-folio#410] [Q&A]
I have been trying to include all references from the `.bib` file without actually citing them in the body of the post.

Is this possible?

**Comment by alshedivat** [comment]
I believe that that's the default behavior of jekyll-scholar, if I'm not mistaken. you just need to add `{% bibliography --file <your-bib-filename> %}` to your post. if you want to only include cited references, then use `{% bibliography --file <your-bib-filename> --cited %}`. See https://github.com/inukshuk/jekyll-scholar#citations.
